client/trino-jdbc/src/test/java/io/trino/jdbc/TestTrinoDriver.java

```diff
@@ -1084,6 +1091,18 @@ public void testUpdatePartialCancel()
         }
     }
 
+    @Test(expectedExceptions = SQLFeatureNotSupportedException.class, expectedExceptionsMessageRegExp = "Multiple open results not supported")
```

Comment:
Don't rearrange existing tests, as that is unrelated to the DNS resolver change

Reply:
I just added a new test, but this is how GH rendered the diff.  I've moved it as the last test in the class.